proxy/src/bind.rs

```diff
@@ -90,7 +102,7 @@ pub struct NormalizeUri<S> {
     inner: S
 }
 
-pub type Service<B> = Binding<B>;
+pub type Service<B> = BoundService<B>;
```

Comment:
Is this type alias still necessary? It was originally added to shorten a really long return type for what's now called "`Stack<B>`". 

Consider either just renaming `BoundService<B>` to `Service<B>` and removing the type alias,  or changing all consumers of this API to refer to `BoundService<B>` (which seems clearer IMHO) and removing the type alias.